setup.py

```diff
@@ -49,12 +49,11 @@ def get_version():
         "Programming Language :: Python",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
```

Comment:
shouldn't we also remove 2.7?

Reply:
> I have not dropped support for Python 2.7.
>
> This library is very simple... it seems silly to drop Python 2.7 support until we can not test it in CI.

Reasons to not deprecate Python 2.7:
1. Technically, our Policy for Minor Runtime deprecation means we do not need to create a new major version of this library if we only remove 3.5 & 3.6. Which is groovy.
2. Our CI still supports Python 2.7
3. This library is very primitive oriented. If there is no reason to drop Python 2.7, I do not think we should.

Reply:
ahhh reading fail 
makes sense - we should track in an issue to remove py27, seems a little odd to still support it since it's been EOL since 2020